events.json

```diff
@@ -1,12 +0,0 @@
----
```

Comment:
Hrmmmm. Site still works without this?

There is an admin page here: https://techwerkers.nl/admin/ and I'm wondering if this file is needed there,

Reply:
Honestly, I yolo-ed. It didn't break my the docker thingy. I don't have access to netlify, so I'm not sure it's something I can check. Can we use the preview: https://deploy-preview-14--twc-site-nl.netlify.app/admin ?